help/Readme.md

```diff
@@ -0,0 +1,116 @@
+- `VirtualMachineName` - The name of the virtual machine to be generated.
+- `AdminUserName` - The administrator username for the virtual machine to be created.
+- `AdminPassword` - The administrator password for the virtual machine to be created.
+- `AzureLocation` - The location where the Azure virtual machine will be provisioned. Example: "eastus"
+
+Fuction will create an Azure VM from a template and generate network resources in Azure to make the VM accessible.
+
+### Additional
+### Variables
+Each packer template includes `variables` group that contains builder variables that are used in image generation.
+Builder variables can be passed to packer via predefined environment variables, or as direct arguments, in case if packer started manually.
+
+#### Builder variables
+- `client_id` - The application ID of the AAD Service Principal. Requires `client_secret`.
+- `object_id` - The object ID for the AAD SP. Will be derived from the oAuth token if empty.
+- `client_secret` - A password/secret registered for the AAD SP.
+- `subscription_id` - The subscription to use.
+- `tenant_id` - The Active Directory tenant identifier with which your `client_id` and `subscription_id` are associated. If not specified, `tenant_id` will be looked up using `subscription_id`.
+- `resource_group` - Resource group under which the final artifact will be stored.
+- `storage_account` - Storage account under which the final artifact will be stored.
+- `location` - Azure datacenter in which your VM will be built.
+- `temp_resource_group_name` - Name assigned to the temporary resource group created during the build. If this value is not set, a random value will be assigned. This resource group is deleted at the end of the build.
+- `private_virtual_network_with_public_ip` - This value allows you to set a `virtual_network_name` and obtain a public IP. If this value is not set and `virtual_network_name` is defined Packer is only allowed to be executed from a host on the same subnet / virtual network.
+- `virtual_network_name` - Use a pre-existing virtual network for the VM. This option enables private communication with the VM, no public IP address is used or provisioned (unless you set `private_virtual_network_with_public_ip`).
+- `virtual_network_resource_group_name` - If `virtual_network_name` is set, this value may also be set. If `virtual_network_name` is set, and this value is not set the builder attempts to determine the resource group containing the virtual network. If the resource group cannot be found, or it cannot be disambiguated, this value should be set.
+- `virtual_network_subnet_name` - If `virtual_network_name is set`, this value may also be set. If `virtual_network_name` is set, and this value is not set the builder attempts to determine the subnet to use with the virtual network. If the subnet cannot be found, or it cannot be disambiguated, this value should be set.
```

Comment:
If `virtual_network_name is set` - too many words in code block. If `virtual_network_name` is set

Reply:
Thanks. Fixed.